tests/unit/api/deleteMarker.js

```diff
@@ -1,10 +1,14 @@
-import { versioning } from 'arsenal';
-
+import async from 'async';
```

Comment:
why create a deleteMarker.js file instead of mutliObjectDelete or Delete?  those are the api's.